app/docs/0.13.x/getting-started/configuring-a-service.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Configuring a Service
+---
+
+# Configuring a Service
+
+<div class="alert alert-warning">
+  <strong>Before you start:</strong>
+  <ol>
+    <li>Make sure you've <a href="/install/">installed Kong</a> &mdash; It should only take a minute!</li>
+    <li>Make sure you've <a href="/docs/{{page.kong_version}}/getting-started/quickstart">started Kong</a>.</li>
+  </ol>
+</div>
+
+In this section, you'll be adding your Service to the Kong layer. This is the first
+step to having Kong manage your API. For purposes of this Getting Started
+guide, we suggest adding the [Mockbin Service][mockbin] to Kong, as Mockbin is
```

Comment:
cannot be "Service". "API" (as before), or "service" if the distinction is clearly made, or even just "Mockbin", but "Service" refers to the Kong entity